src/styles/style.css

```diff
@@ -0,0 +1,6 @@
+ @import './blocks/body.css';
+ @import './blocks/stars.css';
+ @import './blocks/stars-star.css';
+ @import './blocks/stars-stars-star-calm.css';
+ @import './blocks/stars-stars-star-active.css';
```

Comment:
1. no need to create such amount of files, you have 1 block stars, so it would be enogh to move all of the styles with classes `stars__...` to stars.css and remove all other files from blocks
2. you can also create a normalize.css or reset.css and add there styles for body, it's not a block just a reset of default body tag styles